README.md

```diff
@@ -127,7 +126,7 @@ $ npm run yalc-publish
 Use them in your application:
 
 ```bash
-$ yalc add knora-ui
+$ yalc add @knora/ui
```

Comment:
I think we should make an npm script for that. I think it could be something like `npm run yalc-add`. We should also add a script like `npm run yalc-remove` that executes `yalc remove --all`. In the case of `yalc`, it is ok to use a global package.

Reply:
you're right

Reply:
We can also make a separate PR for that, so this PR is not blocked.

Reply:
This should be part of other PR and user story e.g. https://dasch.myjetbrains.com/youtrack/issue/DSP-199